.github/workflows/style.yml

```diff
@@ -0,0 +1,52 @@
+name: Style
+
+on:
```

Comment:
Why don't we run style checks on `push` event too?

Reply:
Techincally it's not possible to push to `main` without creating a PR, because in this repo it's a protected branch, so there's no need to do this

Reply:
In this case why do we use this event in other workflows :thinking:?